src/aura/autocomplete/autocompleteHelper.js

```diff
@@ -71,5 +76,106 @@
         var listComponent = component.find('list');
         $A.util.toggleClass(listComponent, "slds-hide", !visible);
         component.set('v.isListVisible', visible);
+    },
+
+    // Handle the pressing of the arrow key down.
+    // It updates the aria-activedescendant and notify the child elements
+    handleArrowDownKey: function (component, helper) {
+        const listbox = document.querySelector('[role="listbox"]');
+        const elements = component.get("v.items");
+        var currentFocussedElement = listbox.getAttribute('aria-activedescendant');
+        
+        if (currentFocussedElement == null || currentFocussedElement == '') {
+            if (elements[0]) {
+                currentFocussedElement = elements[0].value.Id
+                listbox.setAttribute('aria-activedescendant', currentFocussedElement);
+                helper.fireNewItemFocus(currentFocussedElement);
+                return;
+            } else {
+                var showFooter = component.get('v.showListFooter');
+                if (showFooter) {
+                    helper.fireFocusOnFooter(component, listbox);
+                    return;
+                }
+            }
+            
+        }
+
+        var newFocussedElement = null; 
+        
```

Comment:
Can we remove this blank line?  newFocussedElement is an integral part of the for loop below, so I don't think the blank line helps with understanding what is happening here.